lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -242,6 +259,30 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         }
     }
 
+    saveEditChanges(buttonType: string, event: Event, group?: any) {
```

Comment:
instead of string could you please prepare Enum for all button types? It will be better because when it is just string then it is easy to make typo